opensrp-giz-malawi/src/main/java/org/smartregister/giz/interactor/LoginInteractor.java

```diff
@@ -69,5 +69,11 @@ protected void scheduleJobsImmediately() {
         PullUniqueIdsServiceJob.scheduleJobImmediately(PullUniqueIdsServiceJob.TAG); //need these asap!
         ZScoreRefreshIntentServiceJob.scheduleJobImmediately(ZScoreRefreshIntentServiceJob.TAG);
         ImageUploadServiceJob.scheduleJobImmediately(ImageUploadServiceJob.TAG);
+
+        // This job will not be duplicated but is added here since scheduleJobsPeriodically is only called
+        // after a remote login and therefore might be run too late. scheduleJobsImmediately is called
+        // after both remote login and local login
```

Comment:
This looks like a single comment. Could we use a block comment.

Reply:
Definitely